pertpy/tools/_coda/_base_coda.py

```diff
@@ -2130,21 +2129,23 @@ def my_layout(node):
             tree2.render(save, tree_style=tree_style, units=units)
         if show:
             if not show_leaf_effects:
-                return tree2.render("%%inline", tree_style=tree_style, units=units, w=w, h=h, dpi=dpi)
+                return tree2.render("%%inline", tree_style=tree_style, units=units, w=figsize[0], h=figsize[1], dpi=dpi)
         else:
             if not show_leaf_effects:
                 return tree2, tree_style
 
     def plot_effects_umap(  # pragma: no cover
         self,
-        data: MuData,
+        mudata: MuData,
```

Comment:
```suggestion
        mdata: MuData,
```

adata == AnnData
mdata == Mudata